src/pages/ProfilesExplorerView/components/SceneByVariableRepeaterGrid/domain/sortFavGridItems.ts

```diff
@@ -7,7 +9,7 @@ export const sortFavGridItems: (a: GridItemData, b: GridItemData) => number = fu
   const bIsFav = FavoritesDataSource.exists(FavAction.buildFavorite(b));
 
   if (aIsFav && bIsFav) {
-    return a.label.localeCompare(b.label);
+    return localeCompare(a.label, b.label);
```

Comment:
Wouldn't this be exactly the same? Accroding to [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare):

> The localeCompare() method of [String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String) values returns a number indicating whether this string comes before, or after, or is the same as the given string in sort order. In implementations with [Intl.Collator API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator) support, this method simply calls Intl.Collator.

Reply:
Oh sorry, just read the paragraph below which says:

> When comparing large numbers of strings, such as in sorting large arrays, it is better to create an [Intl.Collator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator) object and use the function provided by its [compare()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/compare) method.

:D

Reply:
No worries :)